### PR TITLE
Create a test image for spooky tests (resolves #177)

### LINKED ISSUE #177: Create image for spooky tests
Add a test image to quay such that it can be used to test the effects of the docker clean actions

### PR DESCRIPTION
**Pull request by arkal**
resolves #177
connected to BD2KGenomics/Toil#987
connected to BD2KGenomics/toil-lib#14

The image will generate a container that creates a file in /data and then sleep for
5 minutes. BD2KGenomics/toil-lib will use this container for testing the utility of
the docker clean functions.

### CODE REVIEW

**Comment by arkal** [comment]
@jvivian can you build this image and tell me when i can pull it from quay?

**Comment by jvivian** [comment]
Jenkins, test this please

**Comment by arkal** [comment]
Jenkins, test this please.

**Comment by jvivian** [comment]
@arkal — Spooky Jenkins builds PR for spooky containers.

**Comment by hannes-ucsc** [comment]
@hannes_ucsc, note to self: revisit status checks page once a build finishes.

**Comment by arkal** [comment]
Jenkins, test this please.

**Comment by hannes-ucsc** [comment]
Jenkins, test this please.

**Comment by arkal** [comment]
@hannes this is ready

File: spooky-test/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Definitions
+runtime_fullpath = $(realpath runtime)
+build_tool = runtime-container.DONE
+name = quay.io/ucsc_cgl/spooky_test
+tag = 1.0.0a--boo
```
Comment (hannes-ucsc) [inline spooky-test/Makefile]: For sake of consistency, this should be the commit ID of the current commit. The other Makefile in this repo have that functionality.

**Comment by arkal** [comment]
@hannes-ucsc i implemented the suggested changes

**Pull request merged**